src/app/detail/ui/TaskEditor.tsx

```diff
@@ -171,6 +175,16 @@ export const TaskEditor = ({
             const url = await getSignedUrlFile(token ?? '', filePayload?.filePath ?? '')
             return url
           }}
+          deleteEditorAttachments={async (url: string) => {
+            const filePath = await getFilePathFromUrl(url)
+            if (filePath) {
+              const payload: ScrapImageRequest = {
+                filePath: filePath,
+                taskId: task_id,
+              }
+              postScrapImage(payload)
+            }
+          }}
```

Comment:
Again, it's better to extract these are into it's own variable and pass that variable as a function rather than declaring them inline.